package.json

```diff
@@ -70,8 +70,8 @@
         "hardhat": "^2.2.1",
         "hardhat-deploy": "0.9.2",
         "husky": "^5.1.3",
-        "prettier": "^2.8.3",
-        "prettier-plugin-solidity": "1.1.1",
+        "prettier": "^2.8.4",
+        "prettier-plugin-solidity": "1.1.2",
```

Comment:
keeping the formatters up-to-date in case there are any improvements for docs n code formatting